docs-v2/integrations/all/getguru.mdx

```diff
@@ -0,0 +1,32 @@
+---
+title: GetGuru
```

Comment:
It looks like the name is just Guru (the URL is getguru.com though). 
<img width="666" alt="Screenshot 2023-12-26 at 22 55 15" src="https://github.com/NangoHQ/nango/assets/2560456/99f3975f-34e6-4d7e-94cc-93bb524f0076">